File: Python/OneLang/StdLib/PackageBundleSource.py
from OneLangStdLib import *
import OneLang.StdLib.PackageManager as packMan

class PackageBundleSource:
    def __init__(self, bundle):
        self.bundle = bundle
    
    def get_package_bundle(self, ids, cached_only):
        raise Error("Method not implemented.")
    
    def get_all_cached(self):
        return self.bundle

File: PHP/src/OneLang/StdLib/PackageBundleSource.php
<?php

namespace OneLang\StdLib\PackageBundleSource;

use OneLang\StdLib\PackageManager\PackageSource;
use OneLang\StdLib\PackageManager\PackageId;
use OneLang\StdLib\PackageManager\PackageBundle;

class PackageBundleSource implements PackageSource {
    public $bundle;
    
    function __construct($bundle) {
        $this->bundle = $bundle;
    }
    
    function getPackageBundle($ids, $cachedOnly) {
        throw new \OneLang\Core\Error("Method not implemented.");
    }
    
    function getAllCached() {
        return $this->bundle;
    }
}
